src/MetricExporter.ts

```diff
@@ -123,11 +125,18 @@ export class DynatraceMetricExporter implements PushMetricExporter {
 						lines = lines.concat(this.serializeCounter(metric));
 						break;
 					case InstrumentType.OBSERVABLE_GAUGE:
+						lines = lines.concat(this.serializeGauge(metric));
+						break;
 					case InstrumentType.UP_DOWN_COUNTER:
-						lines.push(...this.serializeGauge(metric));
+						if (metric.aggregationTemporality === AggregationTemporality.CUMULATIVE) {
```

Comment:
I guess the reason why this check is not inside `serializeGauge` is because in JS we don't have the "monotonicity" right? We rather work with the OTel instrument types? 

Not sure how better/worse would be to still move the check inside `serializeGauge`?

Reply:
That's right. In this case, we are just using the gauge to export non-monotonic counts, so I didn't want to pull that logic into the gauge method. I added another method for updowncounters in 1c2ae39